Desvinculación de cuentas asociadas (#1313)
mercadopago/devsite-docs: guides/online-payments/marketplace/checkout-api/create-marketplace.en.md

    "scope": "offline_access payments write"
}
```
### Unlink accounts

> The unlinking of the token associated with the account is from the [Mercado Pago portal](https://www.mercadopago[FAKER][URL][DOMAIN]/account/security/applications/connections)in: Your profile> Security> Connected apps.

Reviewer: ```suggestion
To unlink a token associated with your account, you can do it from the [Mercado Pago portal](https://www.mercadopago[FAKER][URL][DOMAIN]/account/security/applications/connections) and at **Your profile> Security> Connected apps**.
```
